src/app/content/studyGuides/hooks/loadMore.spec.ts

```diff
@@ -57,24 +57,33 @@ describe('loadMore', () => {
   it('fetches multiple pages across multiple sources', async() => {
     store.dispatch(receiveBook(book));
     store.dispatch(receivePage(page));
+
+    const highlightsCount = {
+      'testbook1-testpage2-uuid': 15,
+      // keep the order of pages in the used fixture
+      // tslint:disable-next-line: object-literal-sort-keys
+      'testbook1-testpage11-uuid': 10,
+      'testbook1-testpage8-uuid': 15,
+    };
+
     store.dispatch(receiveStudyGuidesTotalCounts({
-      'testbook1-testpage1-uuid': {[HighlightColorEnum.Green]: 15},
-      'testbook1-testpage11-uuid': {[HighlightColorEnum.Green]: 5},
-      'testbook1-testpage2-uuid': {[HighlightColorEnum.Green]: 15},
+      'testbook1-testpage2-uuid': {[HighlightColorEnum.Green]: highlightsCount['testbook1-testpage2-uuid']},
+      // tslint:disable-next-line: object-literal-sort-keys
+      'testbook1-testpage11-uuid': {[HighlightColorEnum.Green]: highlightsCount['testbook1-testpage11-uuid']},
+      'testbook1-testpage8-uuid': {[HighlightColorEnum.Green]: highlightsCount['testbook1-testpage8-uuid']},
     }));
     store.dispatch(setDefaultSummaryFilters({locationIds: [
-      'testbook1-testpage1-uuid',
       'testbook1-testchapter1-uuid',
     ]}));
 
     const page1 = createTestHighlights({
-      amount: 15,
-      sourceId: 'testbook1-testpage1-uuid',
+      amount: highlightsCount['testbook1-testpage2-uuid'],
+      sourceId: 'testbook1-testpage2-uuid',
     });
 
     const page2 = createTestHighlights({
-      amount: 5,
-      sourceId: 'testbook1-testpage2-uuid',
+      amount: 20 - page1.length,
```

Comment:
I like the idea of creating `highlightsCount` const
I think `20` should be also in const in this case

Reply:
I named it the same way as the normal variable that is used when calling highlights api. I didn't import it since it could possible change, breaking the tests unnecessarily